src/lib/import-export/export/export-database.ts

```diff
@@ -47,15 +47,25 @@ export async function exportDatabaseToMultipleFiles(
 	}
 
 	const tablesResult = await server.executeWpCliCommand(
-		`sqlite tables --format=csv --require=/tmp/sqlite-command/command.php`
+		`sqlite tables --format=json --require=/tmp/sqlite-command/command.php`
 	);
 	if ( tablesResult.stderr ) {
 		throw new Error( `Database export failed: ${ tablesResult.stderr }` );
 	}
 	if ( tablesResult.exitCode ) {
 		throw new Error( 'Database export failed' );
 	}
-	const tables = tablesResult.stdout.split( ',' );
+
+	let tables;
+
+	try {
+		tables = JSON.parse( tablesResult.stdout );
```

Comment:
After applying https://github.com/Automattic/wp-cli-sqlite-command/pull/9 it worked as expected, producing the output:

```
[
  'wp_users',
  'wp_usermeta',
  'wp_termmeta',
  'wp_terms',
  'wp_term_taxonomy',
  'wp_term_relationships',
  'wp_commentmeta',
  'wp_comments',
  'wp_links',
  'wp_options',
  'wp_postmeta',
  'wp_posts'
]
```

Very smart! 💡